internal/provider/resource_vault_cluster.go

```diff
@@ -1067,11 +1067,35 @@ func flattenMajorVersionUpgradeConfig(config *vaultmodels.HashicorpCloudVault202
 }
 
 func resourceVaultClusterImport(ctx context.Context, d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+	// with multi-projects, import arguments must become dynamic:
+	// use explicit project ID with terraform import:
+	//   terraform import hcp_vault_cluster.test f709ec73-55d4-46d8-897d-816ebba28778:test-vault-cluster
+	// use default project ID from provider:
+	//   terraform import hcp_vault_cluster.test test-vault-cluster
+
 	client := meta.(*clients.Client)
+	projectID := ""
+	clusterID := ""
+	var err error
+
+	if strings.Contains(d.Id(), ":") {
+		idParts := strings.SplitN(d.Id(), ":", 2)
+		// Get the Cluster ID from the input
+		clusterID = idParts[1]
+		// Get the Project ID from the input
+		projectID = idParts[0]
+	} else {
+		// Get the Cluster ID from the input
```

Comment:
I liked the comment format you used elsewhere that shows the ID format 
`// {bucket_name}:{channel_name}`